examples/function/flatmap_stream/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "map-flatmap-stream"
+version = "0.2.4"
+description = ""
+authors = ["Numaflow developers"]
+
+[tool.poetry.dependencies]
+python = "~3.10"
+pynumaflow = "~0.4.0"
+aiorun = "^2022.11.1"
```

Comment:
I don't think they need to add aiorun as their dependency. Pynumaflow already has it :)